### ISSUE TITLE
Feature: capture events for plugged state (onPlugged, onUnplugged)

### ISSUE THREAD

**Issue by sandro-salles** [open-wa/wa-automate-nodejs#480]
Just an idea... it would be very nice to have that event callback, much like we have for onBattery... so immediately we can act upon that change.

Regards

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [1.9.44](https://github.com/open-wa/wa-automate-nodejs/compare/1.9.43...1.9.44) (2020-05-23)

- Release 1.9.44 [`9b8c6e6`](https://github.com/open-wa/wa-automate-nodejs/commit/9b8c6e6f1c6140c92b540b24e266b030e448b9b5)
- feature: implemented `onPlugged` #480 [`fc4558f`](https://github.com/open-wa/wa-automate-nodejs/commit/fc4558f2acfe1a3e5582ccc52444a46c5e2a1507)

**Comment by smashah** [comment]
cool suggestion @sandro-salles 

https://open-wa.github.io/wa-automate-nodejs/classes/client.html#onplugged
